tiny improvements

zssc opened the pull request:
- Added magic comment at the beginning
>Use XeLaTeX to support Chinese better
- Modified the title to make it more formal
- Added 2 new pages to show the pagecount in the template
> I thought the Mr. Blacksmith might forget the pagenumber and tried to add that, then I found `\fancyhead[EL,OR]{\bf\thepage}` in `*.cls`. To avoid confusion I add 2 pages to illustrate it.

zssc committed:
tiny improvements

xjtu-blacksmith commented:
- First two suggestions are excellent and therefore acceptable.
- Page number is **invisible** when:
	- the format of page is `plain` (this will happen on chapter title page), where the header will be omitted;
	- the page number is always placed on the page header.

  Many books are under this kind of rules (e.g. [《普通物理学教程力学》](https://book.douban.com/subject/21993463/) and [《无机化学》](https://book.douban.com/subject/1254433/))，so it should not be considered as a "surprise". The suggestion about `\newpage` is useful, but a little bit "unnatural"; instead, more contents (to extend the span of pages) like figure and table can be added to `demo.tex`, which will also be able to illustrate the layout of page number to users. This will be done in the future.

zssc committed:
add pages and ref

zssc commented:
Thanks for replying!

I can't agree more with your opinion about *unnatural* thing, and the changes are following:
- added some contents in chapter 一元微分学 ;
- added references `ref.bib` (Edited by `JabRef`, a GUI of `BiBTeX` );
- substituted `.` for `。` considering the convention of math books.

I'm wondering if it's too *lengthy* as a template ...

xjtu-blacksmith commented:
> 
> 
> Thanks for replying!
> 
> I can't agree more with your opinion about _unnatural_ thing, and the changes are following:
> 
>     * added some contents in chapter 一元微分学 ;
> 
>     * added references `ref.bib` (Edited by `JabRef`, a GUI of `BiBTeX` );
> 
>     * substituted `.` for `。` considering the convention of math books.
> 
> 
> I'm wondering if it's too _lengthy_ as a template ...

That's not a big problem. Unlike functional packages, demo should contain more contents so that the underlying styles and functions can be seen. Users can leave what they like in the demo, and we do not need to worry about :smile: Something like the demo (or you can say, the user manual) of [ElegantBook](https://github.com/ElegantLaTeX/ElegantBook) is full of contents, but still very useful especially for the new users.

Thanks for your contributing! I'll test your codes then.
